packages/console/src/components/Toast/index.module.scss

```diff
@@ -0,0 +1,19 @@
+@use '@/scss/underscore' as _;
+
+div.toast {
+  padding: _.unit(3) _.unit(4);
+  font: var(--font-body-2);
+  color: var(--color-component-text);
+  box-shadow: 0 2px 8px rgba(0, 0, 0, 8%);
```

Comment:
This value is the same as `Tooltip`'s `box-shadow` property.
Defining it globally would be better.

Reply:
right